webpack/JobInvocationDetail/JobInvocationHostTable.js

```diff
@@ -68,9 +75,20 @@ const JobInvocationHostTable = ({ id, targeting, finished, autoRefresh }) => {
     setAPIOptions: combinedResponse.setAPIOptions,
   });
 
-  const { updateSearchQuery } = useBulkSelect({
+  const { updateSearchQuery: updateSearchQueryBulk } = useBulkSelect({
     initialSearchQuery: urlSearchQuery,
   });
+  const updateSearchQuery = searchQuery => {
+    setParamsAndAPI({
+      ...apiOptions,
+      search: searchQuery,
+    });
+    setAPIOptions({
+      ...apiOptions,
+      params: { search: searchQuery },
```

Comment:
Should include the filter search as well, for example if you select a status and then search the status is not included in the search but still shown as selected